plasma_framework/contracts/src/exits/utils/ExitBounty.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity 0.5.11;
+
+library ExitBounty {
+
+    /**
+     * @notice Returns the Process Exit Bounty size for standard exits
+     */
+    function processStandardExitBountySize() internal view returns (uint256) {
+        return 107000 * tx.gasprice;
```

Comment:
Why that formula? 
It would be nice if we had an explanation in the issue or in a comment here.